tasks/drain.yml

```diff
@@ -0,0 +1,19 @@
+---
+# Ansible tasks to drain a Slurm compute node. Waits for the compute node to be
+# drained for up to a day by default.
+#
+# Variables:
+# - node_to_drain: compute node to drain
+# - drain_timeout: seconds to wait for node to drain, default is 86400.
+
+- name: Drain compute node
+  become: yes
+  command: "scontrol update nodename={{ inventory_hostname }} state=DRAIN reason='maintenance'"
```

Comment:
Just wondering if we should delegate things like this to `openhpc_slurm_control_host`

Reply:
Isn't this what's being done in `tasks/main.yml`?

Reply:
I added the delegation after Stig pointed this out :)